src/Compatibility/ControlGallery/test/Android.UITests/Compatibility.ControlGallery.Android.UITests.csproj

```diff
@@ -66,6 +66,9 @@
       <Project>{57b8b73d-c3b5-4c42-869e-7b2f17d354ac}</Project>
       <Name>Controls.Core</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\..\Controls\src\Xaml\Controls.Xaml.csproj">
```

Comment:
I don't think this is right 

These projects don't use the output from the xaml they only use the test methods that are part of the code behind. Anything that's XAML related gets if/def'd out

What errors were you seeing?

Reply:
it's interesting because on the XF repository we added it to the Android.UITests project but that's it

https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Core.iOS.UITests/Xamarin.Forms.Core.iOS.UITests.csproj#L60

https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Core.Android.UITests/Xamarin.Forms.Core.Android.UITests.csproj#L77

https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Core.Windows.UITests/Xamarin.Forms.Core.Windows.UITests.csproj#L70